lib/net_mosq.c

```diff
@@ -184,6 +170,27 @@ void net__cleanup(void)
 #endif
 }
 
+#ifdef WITH_TLS
+void net__init_tls_engine(void)
```

Comment:
"engine" has a specific meaning in the context of openssl. Could you please rename this to be `net__init_tls()` because it is doing general tls init rather than just the engine.

Reply:
done